test/hooks.test.js

```diff
@@ -74,6 +74,35 @@ test('hooks on static swagger', async t => {
   }
 })
 
+test('onSend hook on static swagger', async t => {
+  const fastify = Fastify()
+  await fastify.register(fastifySwagger, {
+    mode: 'static',
+    specification: {
+      path: './examples/example-static-specification.yaml'
+    }
+  })
+  await fastify.register(fastifySwaggerUi, {
+    uiHooks: {
+      onSend: function (request, reply, payload, done) {
+        const swagger = JSON.parse(payload)
+        swagger.description = 'TestOnSend'
+        const newPayload = JSON.stringify(swagger)
+        done(null, newPayload)
```

Comment:
I would not recommend anybody to do this approach either. What about YAML?
I think the best solution is to do #20

Reply:
Somehow, `yaml` is never be used inside the `swagger-ui`.
I don't know if it is needed inside this library.